### ISSUE TITLE
fails to parse version specifiers with trailing comma

### ISSUE THREAD

**Issue by davidszotten** [astral-sh/uv#1464]
when installing from our devpi mirror, e.g. celery-5.2.2 fails to install. it looks like it's due to a trailing comma in the `requires-python` value. from reading the pep it's slightly unclear to me whether a trailing comma is allowed but `uv` is the first tool that's complained

i added a `dbg!()` because the error message didn't include the offending value (would be a nice addition i think)

```
   0.330188s  40ms  WARN uv_client::registry_client Skipping file for celery: Invalid 'requires-python' value
[crates/uv-client/src/registry_client.rs:632] err = VersionSpecifiersParseError(
    VersionSpecifiersParseError {
        inner: VersionSpecifiersParseErrorInner {
            err: VersionSpecifierParseError {
                kind: MissingOperator,
            },
            line: ">=3.7,",
            start: 6,
            end: 6,
        },
    },
)
```

**Comment by zanieb** [comment]
Hey @davidszotten, which version are you using? I think we may have fixed this already.

**Comment by davidszotten** [comment]
e6c4c77ba10bd54dbf06 (which includes that fix) still has the issue. will make a pr with a failing test (and a suggested fix)

**Comment by charliermarsh** [comment]
Hopefully fixed by #1507?
